### ISSUE TITLE
Typescript attributes incorrect for fields with a defaultValue

### ISSUE THREAD

**Issue by andrewmunro** [sequelize/sequelize-auto#505]
I've noticed that when generating the attributes interface in typescript, fields that have a defaultValue set in the sequelize config will not be marked as optional. Example:

```
export interface WorldTemplateAttributes {
  map: number;
  scriptName: string; <-- should be optional, as the default value below is ""
}

export type WorldTemplatePk = "map";
export type WorldTemplateId = WorldTemplate[WorldTemplatePk];
export type WorldTemplateCreationAttributes = Optional<WorldTemplateAttributes, WorldTemplatePk>;

export class WorldTemplate extends Model<WorldTemplateAttributes, WorldTemplateCreationAttributes> implements WorldTemplateAttributes {
  map!: number;
  scriptName!: string;


  static initModel(sequelize: Sequelize.Sequelize): typeof WorldTemplate {
    WorldTemplate.init({
    map: {
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true
    },
    scriptName: {
      type: DataTypes.STRING(128),
      allowNull: false,
      defaultValue: "", <-- Default value set
      field: 'ScriptName'
    }
  }, {
    sequelize,
    tableName: 'world_template',
    timestamps: false
  });
  return WorldTemplate;
  }
}
```

This makes it an inconvenience when constructing new objects with lots of fields with defaults.

**Comment by steveschmitt** [comment]
Good catch.  Will be fixed in next release.

**Comment by steveschmitt** [comment]
Fixed in 0.8.1
